data/lib/src/network/linshare_http_client.dart

```diff
@@ -128,6 +131,25 @@ class LinShareHttpClient {
     return resultJson.map((data) => _getDynamicAutoCompleteResult(data)).toList();
   }
 
+  Future<List<WorkGroupNodeDto>> getWorkGroupChildNodes(
+    SharedSpaceId sharedSpaceId, [
+    WorkGroupNodeId parentId,
+  ]) async {
+    final endpointPath = Endpoint.sharedSpaces
+        .withPathParameter('/${sharedSpaceId.uuid}')
+        .withPathParameter(Endpoint.nodes)
+        .generateEndpointPath();
+
+    final queryParameters = parentId != null ? {'parent': parentId.uuid} : null;
+
+    final List nodesJsonResult = await _dioClient.get(
+      endpointPath,
+      queryParameters: queryParameters,
+    );
+
+    return nodesJsonResult.map((data) => _convertToWorkGroupNodeChild(data)).toList();
```

Comment:
Can you sort that list in modification Date for default (the latest file in the top of list), right now, when make a change to file (upload), we don't know where it is?

Reply:
Fixed it. Plz help me verify